sbol2/document.py

```diff
@@ -47,6 +47,42 @@
 
 import requests
 
+Config.SBOL_DATA_MODEL_REGISTER = {
```

Comment:
Why not move this to the `Config` class?

Reply:
I tried that initially.  In order to populate the dict with the builders, you have to move all of the imports in document.py into config.py.  That actually turns into a tangled nest and issues with circular imports.

Reply:
Ah yes. Ok, good workaround.